test/benchmark/README.md

```diff
@@ -97,6 +98,7 @@ To add a new benchmark to the Asternias Continuous Integration (CI) system, foll
     - `benchmark_type`: This parameter defines the type of benchmark to be executed. The default value is `guest_only`. The available options include `guest_only`, and `host_guest`.
       - `guest_only`: Use this option when the benchmark is intended solely for the guest environment.
       - `host_guest`: Choose this option when the benchmark involves both the host and guest environments. When using this option, you will need to define your own `host.sh` and `bench_runner.sh` scripts to handle the host-side operations and benchmark execution.
+    - `aster_scheme`: Specify the scheme used in Asterinas. The optional values, e.g., `iommu`, are aligned with the `SCHEME` parameter in `asterinas/Makefile`.
```

Comment:
I guess that we will retire this option soon since the memory safety of Asterinas requires enabling IOMMU. So technically, we should have enabled IOMMU by default and for ALL benchmarks. It is ok to add this option for now.